docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,284 @@
+      "\n",
+      "And this DataBin has attributes: dict_keys(['evs', 'stds', 'ensemble_standard_error'])\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f'The result of the submitted job had {len(result)} PUB and has a value:\\n {result}\\n')\n",
+    "print(f'The associated PubResult of this job has the following data:\\n {result[0].data}\\n')\n",
+    "print(f'And this DataBin has attributes: {result[0].data.keys()}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a93948fa-35f6-4354-9df7-510a02998f79",
+   "metadata": {},
+   "source": [
+    "### Sampler output\n",
+    "\n",
+    "\n",
+    "The Sampler primitive outputs job results in a similar format, with the exception that each `DataBin` will contain one or more `BitArray` objects that store the measured probability distribution(s). The number and attribute label for each bit array object depends on how the `ClassicalRegisters` were defined for the circuit being executed. The measurement data from these `BitArrays` can then be processed into a dictionary with key-value pairs corresponding to each bitstring measured (for example, '1011001') and the number of times (or counts) it was measured.\n",
+    "\n",
+    "For example, a circuit that has measurement instructions added by the [`QuantumCircuit.measure_all()`](../api/qiskit/qiskit.circuit.QuantumCircuit#measure_all) function possesses a classical register with the label *'meas'*. After execution, a count data dictionary can be created by executing:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea9cd691-58d2-4612-86b1-e0b58551b352",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add measurement instructions to the example circuit\n",
+    "circuit.measure_all()\n",
+    "\n",
+    "# Transpile the circuit\n",
+    "pm = generate_preset_pass_manager(optimization_level=1, backend=backend)\n",
+    "transpiled_circuit = pm.run(circuit)\n",
+    "\n",
+    "# Create a PUB for the Sampler primitive using the same parameters defined earlier\n",
+    "sampler_pub = (transpiled_circuit, params)\n",
+    "\n",
+    "\n",
+    "sampler = Sampler(mode=backend)\n",
+    "job = sampler.run([sampler_pub])\n",
+    "result = job.result()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a5160c3f-e977-47c0-a351-66e2cb544c22",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sampler Data is: DataBin(meas=BitArray(<shape=(100,), num_shots=4096, num_bits=2>), shape=(100,))\n",
+      "Its keys are: dict_items([('meas', BitArray(<shape=(100,), num_shots=4096, num_bits=2>))])\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'11': 115808, '10': 104963, '01': 89334, '00': 99495}"
```

Comment:
nitpick: feels like this is missing some text? the other data has a little explanation like "sampler data is:..."

Reply:
Hopefully I addressed this! Feel free to give it another read.